makefile

```diff
@@ -8,7 +8,8 @@ node_modules:
 	npm install prettier clang-format css-validator html-validate eslint eslint-config-google
 
 devprep: 
-	cp src/main/webapp/vendor/startbootstrap/index.html src/main/webapp/index.html
+	cp src/main/webapp/vendor/startbootstrap/index.jsp src/main/webapp/index.jsp
```

Comment:
do we still need to do this? especially for planner.jsp, I don't think it's necessary since we'll be dynamically serving them.